src/main/rx/sbus.c

```diff
@@ -155,13 +160,26 @@ bool sbusInit(const rxConfig_t *rxConfig, rxRuntimeConfig_t *rxRuntimeConfig)
 {
     static uint16_t sbusChannelData[SBUS_MAX_CHANNEL];
     static sbusFrameData_t sbusFrameData;
+    static uint32_t sbus_baud_rate;
 
     rxRuntimeConfig->channelData = sbusChannelData;
     rxRuntimeConfig->frameData = &sbusFrameData;
     sbusChannelsInit(rxConfig, rxRuntimeConfig);
 
     rxRuntimeConfig->channelCount = SBUS_MAX_CHANNEL;
-    rxRuntimeConfig->rxRefreshRate = 11000;
+
+    if (rxConfig->serialrx_provider == SERIALRX_DJI_HDL_7MS)
```

Comment:
Please put braces for blocks on the same line. See https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md for the coding style guidelines.